src/test/java/org/openRealmOfStars/player/player/PlayerInfoTest.java

```diff
@@ -0,0 +1,364 @@
+        PlayerInfo human = new PlayerInfo(race);
+        TechList techList = human.getTechList();
+        Tech[] tech = techList.getList();
+        ShipStat[] statList = human.getShipStatList();
+        ShipStat expectedStat = new ShipStat(ShipGenerator.createScout(human));
+
+        assertEquals(TechType.Combat, tech[0].getType());
+        assertEquals(1, tech[0].getLevel());
+        assertEquals(TechType.Defense, tech[1].getType());
+        assertEquals(1, tech[1].getLevel());
+        assertEquals("Colony", tech[2].getName());
+        assertEquals(1, tech[2].getLevel());
+        assertEquals("Scout Mk1", tech[3].getName());
+        assertEquals(1, tech[3].getLevel());
+        assertEquals("Ion drive Mk1", tech[4].getName());
+        assertEquals(1, tech[4].getLevel());
+        assertEquals("Fission source Mk1", tech[5].getName());
+        assertEquals(1, tech[5].getLevel());
+        assertEquals(expectedStat.toString(), statList[0].toString());
+        expectedStat = new ShipStat(ShipGenerator.createColony(human, false));
+        assertEquals(expectedStat.toString(), statList[1].toString());
+    }
+
+    /*
+     * input : SpaceRace.SPORKS output : PlayerInfo's techList ={ one random
+     * weapon in Combat TechType level 1, one random weapon in Combat TechType
+     * level 1, one random shield or armor in Defense Type level 1, Hulltech
+     * level 1 Colony, Hulltech level 1 Scout Mk1, PropulsionTech level 1 Ion
+     * drive Mk1, PropulsionTech level 1 Fission source Mk1 }, PlayerInfo's
+     * ShipStatList = { scout have random weapon in two weapons, random armor,
+     * Scout Mk1 Hull, Ion drive Mk1, Fission source Mk1 colony have Colony
+     * Hull, Ion drive Mk1, Fission source Mk1 } purpose : test PlayerInfo
+     * constructor Sporks
+     */
+    @Test
+    public void testPlayerInfoSporks() {
+        SpaceRace race = SpaceRace.SPORKS;
+        PlayerInfo human = new PlayerInfo(race);
+        TechList techList = human.getTechList();
+        Tech[] tech = techList.getList();
+        ShipStat[] statList = human.getShipStatList();
+
+        assertEquals(TechType.Combat, tech[0].getType());
+        assertEquals(1, tech[0].getLevel());
+        assertEquals(TechType.Combat, tech[1].getType());
+        assertEquals(1, tech[1].getLevel());
+        assertEquals(TechType.Defense, tech[2].getType());
+        assertEquals(1, tech[2].getLevel());
+        assertEquals("Colony", tech[3].getName());
+        assertEquals(1, tech[3].getLevel());
+        assertEquals("Scout Mk1", tech[4].getName());
+        assertEquals(1, tech[4].getLevel());
+        assertEquals("Ion drive Mk1", tech[5].getName());
+        assertEquals(1, tech[5].getLevel());
+        assertEquals("Fission source Mk1", tech[6].getName());
+        assertEquals(1, tech[6].getLevel());
+
+        ShipStat expectedStat = new ShipStat(ShipGenerator.createScout(human));
+        assertEquals(expectedStat.toString(), statList[0].toString());

```

Comment:
Here is link to Travis error log: https://travis-ci.org/tuomount/Open-Realms-of-Stars/builds/219981697